components/director/internal/tenantmapping/model.go

```diff
@@ -0,0 +1,89 @@
+package tenantmapping
+
+import (
+	"net/http"
+
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/pkg/errors"
+)
+
+// AuthIDSrc wraps possible sources of auth ID like token, username and certificate
+type AuthIDSrc string
+
+// AuthIDRefObjectType wraps possible options for the origin object represented by the auth ID
+type AuthIDRefObjectType string
+
+func sysAuthToAuthIDRefObjectType(sysAuth model.SystemAuth) AuthIDRefObjectType {
+	if sysAuth.AppID != nil {
+		return ApplicationRefObject
+	}
+
+	if sysAuth.RuntimeID != nil {
+		return RuntimeRefObject
+	}
+
+	if sysAuth.IntegrationSystemID != nil {
+		return IntegrationSystemRefObject
+	}
+
+	return ""
+}
+
+const (
+	ApplicationRefObject       AuthIDRefObjectType = "Application"
```

Comment:
in `model/system_auth.go` we define similar consts, can we reuse it?

Reply:
Similar, but its context is different. It is missing the User as reference object which as System Auth does not exist